ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__ibmcloud-4.18-nightly-x86.yaml

```diff
@@ -53,6 +53,40 @@ tests:
     - chain: openshift-qe-ingress-perf
     workflow: openshift-qe-installer-ibmcloud-ipi-ovn
   timeout: 5h0m0s
+- always_run: false
+  as: node-density-heavy-etcd-fips-24nodes
+  steps:
+    allow_skip_on_success: true
+    cluster_profile: gcp-qe
+    env:
+      COMPUTE_NODE_REPLICAS: "24"
+      PROFILE_TYPE: reporting
+    test:
+    - chain: openshift-qe-node-density-heavy
+    workflow: openshift-qe-installer-gcp-ipi-ovn-etcd-encryption-fips
```

Comment:
think you want the **openshift-qe-installer-ibmcloud-ipi-ovn** workflow here so you get an ibm cloud openshift cluster. Same with all the jobs below